src/test/java/BaseTest.java

```diff
@@ -1,18 +1,46 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.firefox.FirefoxDriver;
-import org.openqa.selenium.safari.SafariDriver;
+import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
 
-
+import java.time.Duration;
 public class BaseTest {
-
     @BeforeSuite
-    static void setupClass() {
+    public void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
+        WebDriver driver;
 
+    @BeforeMethod
+    public void setUpBrowser(){
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));}
+    public void openLoginURL() {
+        WebDriver driver = new ChromeDriver();
```

Comment:
**Please, remove code redundancies**

**Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )**



```
  public void openLoginURL() {
    String url = "https://bbb.testpro.io/";
    driver.get(url);
  }
```